client/src/test/utils/utils.test.js

```diff
@@ -0,0 +1,55 @@
+import { getFullname, makeQueryStringTransformable } from '../../utils/utils';
+
+it('Shoud check if getFullname util works', () => {
+  expect(getFullname({ firstName: 'Jack', lastName: 'John' })).toEqual(
+    'Jack John'
+  );
+  expect(getFullname({ firstName: 'Jack' })).toEqual('Jack');
+  expect(getFullname({ firstName: '', lastName: '' })).toEqual('-');
```

Comment:
why do we expect fullname to be '-' if firstname and lastname is empty?

Reply:
Because the util method is written this way.

Reply:
Don't you think if thats the case we need to fix util method instead of hack test?